instruments/technical_docs_for_high_risk_ai.yaml

```diff
@@ -300,18 +299,98 @@ tasks:
+    urn: "urn:nl:aivt:ir:td:q:1.0:8.4"
+    labels:
+      - key: topic
+        value: "EU-conformiteitsverklaring"
+    type: ["statement"]
+    suggested_roles: [ "Projectleider", "Aanbieder"]
+    lifecycle: [ "Validatie" ]
+    links: [ ]
+    template: *assessmentcard_template
+  - question: "Indien een AI-systeem de verwerking van persoonsgegevens met zich meebrengt, voldoet dat AI-systeem aan de Verordeningen (EU) 2016/679 en (EU) 2018/1725 en Richtlijn (EU) 2016/680."
+    urn: "urn:nl:aivt:ir:td:q:1.0:8.5"
+    labels:
+      - key: topic
+        value: "EU-conformiteitsverklaring"
+    type: ["statement"]
+    suggested_roles: [ "Projectleider", "Aanbieder"]
+    lifecycle: [ "Validatie" ]
+    links: [ ]
+    template: *assessmentcard_template
+  - question: "Referenties van relevante gebruikte geharmoniseerde normen of andere gemeenschappelijke specificaties waarop de conformiteitsverklaring betrekking heeft."
+    urn: "urn:nl:aivt:ir:td:q:1.0:8.6"
+    labels:
+      - key: topic
+        value: "EU-conformiteitsverklaring"
+    type: ["open_question"]
+    suggested_roles: [ "Projectleider", "Aanbieder"]
+    lifecycle: [ "Validatie" ]
+    links: [{"text": "Vraag 7 van technical docs","urn": "urn:nl:aivt:ir:td:q:1.0:7"} ]
+    template: *assessmentcard_template
+  - question: "De naam en het identificatienummer van de aangemelde instantie, een beschrijving van de uitgevoerde conformiteitsbeoordelingsprocedure en identificatie van het afgegeven certificaat."
+    urn: "urn:nl:aivt:ir:td:q:1.0:8.7"
+    labels:
+      - key: topic
+        value: "EU-conformiteitsverklaring"
+    type: ["open_question"]
+    suggested_roles: [ "Projectleider", "Aanbieder"]
+    lifecycle: [ "Validatie" ]
+    links: [ {"systemcard_path": ".owners/oin", "version": ">0.1a1"}, {"systemcard_path": ".owners/organization", "version": ">0.1a1"},]
+    template: *assessmentcard_template
+  - signing_task: "Handtekening voor EU-conformiteitsverklaring."
+    urn: "urn:nl:aivt:ir:td:st:1.0:8.8"
+    labels:
+      - key: topic
+        value: "EU-conformiteitsverklaring"
+    required_roles: [ "Aanbieder", "Projectleider"] # TO DO: ensure that both roles need to sign the conformity assessment
+    lifecycle: [ "Validatie" ]
+    links: [ ]
+    depends_on: [{"text":"Punt 1 conformiteitsverklaring", "urn": "urn:nl:aivt:ir:td:q:1.0:8.1"}, {"text":"Punt 2 conformiteitsverklaring", "urn": "urn:nl:aivt:ir:td:q:1.0:8.2"}, {"text":"Punt 3 conformiteitsverklaring", "urn": "urn:nl:aivt:ir:td:q:1.0:8.3"}, {"text":"Punt 4 conformiteitsverklaring", "urn": "urn:nl:aivt:ir:td:q:1.0:8.4"}, {"text":"Punt 5 conformiteitsverklaring", "urn": "urn:nl:aivt:ir:td:q:1.0:8.5"}, {"text":"Punt 6 conformiteitsverklaring", "urn": "urn:nl:aivt:ir:td:q:1.0:8.6"}, {"text":"Punt 7 conformiteitsverklaring", "urn": "urn:nl:aivt:ir:td:q:1.0:8.7"}, {"text":"Punt 8 conformiteitsverklaring", "urn": "urn:nl:aivt:ir:td:q:1.0:8.8"}]
+    encryption:
```

Comment:
Hmm I wouldn't add the encryption way in the task itself, I would just as the other questions, specify the type as ["signature"] and just have an open field where signers can put in their signature hash

Reply:
I have added the extra fields as optional instead of mandatory